kernel-python/declarativewidgets/widget_visualizations.py

```diff
@@ -2,13 +2,32 @@
 # Distributed under the terms of the Modified BSD License.
 
 from IPython.core.display import display, HTML
+import pandas
 
 def explore(df, channel='default', selection_var=None):
-    """Renders the urth-viz-explorer widget to the user output"""
+    """
+    Renders the urth-viz-explorer widget to the user output
+    If pandas.DataFrame assign with unique name to user namespace else use what was passed in the string
+
+    Parameters
+    ----------
+    df              The dataframe itself or the string representation of the variable
+    channel         The channel to bind to defaulted to default
+    selection_var   The selection variable by default not used/applied
+    """
+
+    explore_df = "unique_explore_df_name"
```

Comment:
If this name is not unique... then several `explore` functions in the same notebook would clubber each other.